python3: can't open file '/usr/home/rest/DashBoard/main/scripts/add_val_DB.py': [Errno 2] No such file or directory
Traceback (most recent call last):
  File "/usr/home/rest/DashBoard/dashboard/main/scripts/add_val_DB.py", line 8, in <module>
    import pytz
ModuleNotFoundError: No module named 'pytz'
